### DISCUSSION TITLE
為什麼記憶體是4+4>8?

### DISCUSSION THREAD

**Discussion by Ben6uo** [1101-GS4719-C/Python-Community-GS4719-C#388] [General]
應該有很多人在看電腦推薦配置時都會看到在記憶體那裡看到 4G+4G/8G+8G  而不是直接一條8G/16G吧
那是因為在有2條ram的時候會觸發雙通道  就是在傳輸數據的時有2條通道一起傳輸 那樣速度會比只有一條ram快一倍  
那如果你的ram是4G+8G的話  只會有4G+4G是雙通道  之後的4G是單通道
不過插法都是有條件的

**Comment by pekoranoinu** [comment]
我也有同樣的疑問，大佬請解答。
